### DISCUSSION TITLE
关于数学公式字体的问题

### DISCUSSION THREAD

**Discussion by Iris-Tien** [stone-zeng/fduthesis#259] [Q&A] [answered]
想让论文中的字体 西文是Times New Roman，数学公式是lm 

现有的模板中，如果选择times*，则数学公式的字体如图1，字体偏粗，与向量的区分度不是很大，感觉不好看，而且一般的论文用的也不是这种字体
![466340684797afe32a7181f71b98baf](https://user-images.githubusercontent.com/106738274/173658251-277d2ef1-3f2d-4ba4-bd90-b20e8859486b.png)

但如果选择lm，则数学公式会变好看，但是所有的加粗都不起作用了，看不出那些是向量；而且西文字体也不是 新罗马了。。。如图
![a163d6660ee9853693e95691476219a](https://user-images.githubusercontent.com/106738274/173658766-e8f5f433-6217-4a92-8bbd-816e8d2021a0.png)

我想要的是这种一般的texstudio默认的，西文是新罗马，数学公式是有加粗作用的lm，这样的数学公式更加美观，并且也与绝大多数权威论文所用字体一致。
![984c47394181a9f5c077f4bd643a054](https://user-images.githubusercontent.com/106738274/173659161-a1dd3b19-2cea-420a-8b35-08914e4a9cf3.png)

在现有的模板中是否能达到这样的效果呢？如果能，如何操作？如果不能，今后的模板中能增加这样的选项吗？谢谢~

**Comment by stone-zeng** [comment, marked as answer]
可以在 `\fdusetup` 中设置 `font=times*` 之后，在导言区中加上

```tex
\setmathfont{latinmodern-math.otf}
\setmathrm{latinmodern-math.otf}
```

加粗在本模板中需使用 `\symbf` 命令（直立的加粗使用 `\symbfup`）。

我并不是很建议为公式和正文设置不同的字体，当然，取决于个人选择（以及学校要求）。

**Reply by sikouhjw** [reply]
我建议加个 `math-font` 选项，跟 `font` 独立

**Reply by Iris-Tien** [reply]
哇！回复地好快，超级感动~回复中的方法用过了！切实可行，达到了预期目标，非常感谢！（另外也建议像楼上说的那样把西文和公式的字体设定分开，现在这个模板被学校很多不同专业的同学安利使用，像力学系和数学系这些公式码得很多的就要求字体用lm，所以希望作者有空的时候可以考虑修改一下这个地方~谢谢~
